### ISSUE TITLE
Avoid joining multiline Ruby to not anger RuboCop

### ISSUE THREAD

**Issue by danieldiekmeier** [sds/haml-lint#361]
Thanks for building HAML-Lint, it really helped me clean up the templates in our Rails app!

I have a file where I want to break inside a hash to make my file more readable. Simplified example:

```haml
.example{class: class_names({
  optional: maybe?
})}
```

This works and is valid Haml, but my RuboCop configuration is set to `Layout/SpaceInsideHashLiteralBraces: no_space`, which makes RuboCop angry because  HAML-Lint joins the lines of the hash but keeps spaces around the hash braces.

To make sure I understood this right, I added a test case to `spec/haml_lint/ruby_extractor_spec.rb`:

```ruby
context 'with a tag with hash attributes containing a hash' do
  let(:haml) { <<-HAML }
    %tag{class: class_names({
      optional: maybe?
    })}
  HAML

  its(:source) { should == normalize_indent(<<-RUBY).rstrip }
    {}.merge(class: css_names({
      optional: maybe?
    }))
    _haml_lint_puts_0 # tag
    _haml_lint_puts_1 # tag/
  RUBY
end
```

And indeed, the diff is:

```diff
-{}.merge(class: css_names({
-  optional: maybe?
-}))
+{}.merge(class: class_names({ optional: maybe? }))
 _haml_lint_puts_0 # tag
 _haml_lint_puts_1 # tag/
```

Of course, if we were to simply remove the spaces, _my_ RuboCop would be happier, but if RuboCop was configured the other way (`space`), it would be angry as well.

My suggestion would be to _not_ join the lines of a multiline hash. Indentation [already does not matter ](https://github.com/sds/haml-lint/blob/1231b394bcc2b6c9c47a7a18074b43b258a0872e/config/default.yml#L90), so I think this could be an option?

The same change would probably need to be made for **parentheses**, because RuboCop dislikes this HAML for the same reason:

```haml
.example{class: class_names(
  {optional: maybe?}
)}
```

I could give this a go if you are open to the change. Thanks for your consideration!

**Comment by sds** [comment]
Very open to a contribution that addresses! Thanks for the detailed explanation.

**Comment by danieldiekmeier** [comment]
Closing this because I created a PR which was merged! Thanks again!
